.github/workflows/check.yml

```diff
@@ -264,3 +264,22 @@ jobs:
       with:
         name: test logs
         path: test-logs
+
+  # This is a workaround to CodeQL not reporting status to the
+  # merge queue.
+  check_codeql_status:
```

Comment:
I'm a bit confused. Shouldn't this replace the existing Perform CodeQL Analysis step?

Reply:
Still a bit too much magic for me to fully understand :/